Mlem/App/Constants/Colors/MonochromePalette.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MonochromePalette.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-05-06.
+//
+
+import Foundation
+import SwiftUI
+
+struct MonochromePalette: PaletteProviding {
+    let primary = Color.primary
+    let background = Color(UIColor.systemBackground)
+    let secondaryBackground = Color(UIColor.secondarySystemBackground)
+    let tertiaryBackground = Color(UIColor.tertiarySystemBackground)
+    let accent = Color.black
+    let uiAccent = UIColor.black
+    
+    let upvote = Color.black
+    let downvote = Color.black
+    let save = Color.black
```

Comment:
These should be `Color.primary` so that it works correctly in dark mode. This will, however, cause the interaction buttons to appear completely white in dark mode - w'll probably need a separate `selectedInteractionBarItem` color to fix this.

```suggestion
    let accent = Color.primary
    let uiAccent = UIColor.label
    
    let upvote = Color.primary
    let downvote = Color.primary
    let save = Color.primary
```